config.json

```diff
@@ -18,6 +18,17 @@
 	    ]
 	},
 	"propagatedmount": "/mnt",
+	"mounts": [
+		{
+		"name": "KeyPath",
+		"description": "Path to SSH keys",
+		"source": "/tmp",
+		"destination": "/tmp/sshfs",
```

Comment:
@stevvooe:  Hm - I suppose one solution is to mount all of `/` into the plugin container, and allow the user to specify the `.ssh` directory to use for the credentials.  I think the goal was to try to not have the plugin container mount everything, though.